### PR TITLE
DOCS-5485 - Guide for service account impersonation and automatic project discovery

### PR DESCRIPTION
**Pull request by cswatt**
hold for merge

### CODE REVIEW

File: content/en/integrations/guide/service-account-impersonation.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    - link: 'https://cloud.google.com/iam/docs/granting-changing-revoking-access'
+      tag: 'Google Cloud'
+      text: 'Manage access to projects, folders, and organizations'
+---
+
+Instead of using static credentials in private key files, you can use [service account impersonation][1] and automatic project discovery to integrate Datadog with [Google Cloud][2].
+
+This solution enables you to monitor all projects visible to a service account by assigning the required roles in the relevant projects. You can assign these roles to projects individually, or you can configure Datadog to monitor groups of projects by assigning these roles at the organization or folder level. Assigning roles in this way allows Datadog to automatically discover and monitor all projects in the given scope, including any new projects that may be added to the group in the future. 
+
+### Prerequisites
```
Comment (rtrieu) [inline content/en/integrations/guide/service-account-impersonation.md:17]:
```suggestion
## Prerequisites
```

File: content/en/integrations/guide/service-account-impersonation.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+* Service account impersonation and automatic project discovery rely on having certain roles and APIs enabled to monitor projects. Before you start, ensure the following APIs are enabled for the projects you want to monitor:
+  * [Cloud Resource Manager API][3]
+  * [Google Cloud Billing API][4]
+  * [Cloud Monitoring API][5]
+  * [Compute Engine API][6]
+  * [Cloud Asset API][7]
+  * [IAM API][8]
+
+### Setup
```
Comment (rtrieu) [inline content/en/integrations/guide/service-account-impersonation.md:29]:
```suggestion
## Setup
```

File: content/en/integrations/guide/service-account-impersonation.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      text: 'Manage access to projects, folders, and organizations'
+---
+
+Instead of using static credentials in private key files, you can use [service account impersonation][1] and automatic project discovery to integrate Datadog with [Google Cloud][2].
+
+This solution enables you to monitor all projects visible to a service account by assigning the required roles in the relevant projects. You can assign these roles to projects individually, or you can configure Datadog to monitor groups of projects by assigning these roles at the organization or folder level. Assigning roles in this way allows Datadog to automatically discover and monitor all projects in the given scope, including any new projects that may be added to the group in the future. 
+
+### Prerequisites
+
+* If your organization restricts identities by domain, you must add Datadog's customer identity as an allowed value in your policy. Datadog's customer identity: `C0147pk0i`
```
Comment (rtrieu) [inline content/en/integrations/guide/service-account-impersonation.md:19]:
```suggestion
* If your organization restricts identities by domain, you must add Datadog's customer identity as an allowed value in your policy. Datadog's customer identity is: `C0147pk0i`
```

File: content/en/integrations/guide/service-account-impersonation.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+4. Copy your Datadog principal and keep it for the next section.
+   {{< img src="integrations/guide/service_account_impersonation/principal-2.png" alt="Datadog interface, showing the 'Add New GCP Account' flow. The first step, 'Add Datadog Principal to Google,' features a text box where a user can generate a Datadog Principal and copy it to their clipboard. The second step, 'Add Service Account Email,' features a text box that the user can complete in section 3." style="width:70%;">}}
+   Keep this window open for the [next section](#3-complete-the-integration-setup-in-datadog).
+5. In [Google Cloud console][9], under the **Service Acounts** menu, find the service account you created in the [first section](#1-create-your-google-cloud-service-account).
+6. Go to the **Permissions** tab and click on **Grant Access**.
+   {{< img src="integrations/guide/service_account_impersonation/grant-access.png" alt="Google Cloud console interface, showing the Permissions tab under Service Accounts." style="width:70%;">}}
+7. Paste your Datading principal into the **New principals** text box.
```
Comment (rtrieu) [inline content/en/integrations/guide/service-account-impersonation.md:57]:
```suggestion
7. Paste your Datadog principal into the **New principals** text box.
```

File: content/en/integrations/guide/service-account-impersonation.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+8. Assign the role of **Service Account Token Creator** and click **Save**.
+   {{< img src="integrations/guide/service_account_impersonation/add-principals-blurred.png" alt="Google Cloud console interface, showing an 'Add principals' box and an 'Assign roles' interface." style="width:70%;">}}
+
+**Note**: If you previously configured access using a shared Datadog principal, you can revoke the permission for that principal after you complete these steps.
+
+### 3. Complete the integration setup in Datadog
+
+1. In your Google Cloud console, navigate to the **Service Account** > **Details** tab. There, you can find the email associated with this Google service account. It resembles `<sa-name>@datadog-sandbox.iam.gserviceaccount.com`.
+2. Copy this email.
+3. Return to the integration configuration tile in Datadog (where you coppied your Datadog principal in the [previous section](#2-add-the-datadog-principal-to-your-service-account)).
```
Comment (rtrieu) [inline content/en/integrations/guide/service-account-impersonation.md:67]:
```suggestion
3. Return to the integration configuration tile in Datadog (where you copied your Datadog principal in the [previous section](#2-add-the-datadog-principal-to-your-service-account)).
```

File: content/en/integrations/guide/service-account-impersonation.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+4. In the box under **Add Service Account Email**, paste the email you previously coppied.
```
Comment (rtrieu) [inline content/en/integrations/guide/service-account-impersonation.md:68]:
```suggestion
4. In the box under **Add Service Account Email**, paste the email you previously copied.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some minor edits, but otherwise looks good!

**Comment by cswatt** [comment]
stale, added to dogweb instead

**Pull request closed without merging**